ledger/storage/in_mem_storage.go

```diff
@@ -0,0 +1,54 @@
+package storage
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/ledger/common/hash"
+)
+
+type InMemStorage struct {
+	store map[hash.Hash][]byte
+}
+
+func NewInMemStorage() *InMemStorage {
```

Comment:
This is a dummy implementation of the storage layer, will be replaced by an on-disk key-value store implementation.